app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -606,9 +606,6 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
   }
 
   label {
-    @extend .crayons-field;
-    @extend .crayons-field__label;
```

Comment:
This was adding `margin-top` to the a tag.

Reply:
I see the problem you mention with the margin-top, but removing these `@extend`s alters the sizing a little, and loses the link with our design system scss

If you reinstate these lines, you should be able to get the same effect while keeping the other styles with

```scss
    @extend .crayons-field;
    @extend .crayons-field__label;

    a {
      margin-top: 0;
    }
```

Reply:
I have made the above changes now.